cmd/watchdog/main.go

```diff
@@ -183,19 +206,25 @@ func downloadGTFSBundles(servers []models.ObaServer, cacheDir string, logger *sl
 }
 
 // refreshGTFSBundles periodically downloads GTFS bundles at the specified interval.
-func refreshGTFSBundles(servers []models.ObaServer, cacheDir string, logger *slog.Logger , interval time.Duration) {
+func refreshGTFSBundles(servers []models.ObaServer, cacheDir string, logger *slog.Logger , interval time.Duration , reporter *report.Reporter) {
 	for {
 		time.Sleep(interval)
-		downloadGTFSBundles(servers, cacheDir, logger)
+		downloadGTFSBundles(servers, cacheDir, logger , reporter)
 	}
 }
 
 // refreshConfig periodically fetches remote config and updates the application servers.
-func refreshConfig(configURL, configAuthUser, configAuthPass string, app *application, logger *slog.Logger , interval time.Duration) {
+func refreshConfig(configURL, configAuthUser, configAuthPass string, app *application, logger *slog.Logger , interval time.Duration , reporter *report.Reporter) {
 	for {
 		time.Sleep(interval)
-		newServers, err := loadConfigFromURL(configURL, configAuthUser, configAuthPass)
+		newServers, err := loadConfigFromURL(configURL, configAuthUser, configAuthPass , reporter)
 		if err != nil {
+			reporter.ReportErrorWithSentryOptions(err, report.SentryReportOptions{
+				Tags: map[string]string{
```

Comment:
you're creating these `map[string]string` types a lot here for maps with a single key/value pair. I recommend creating a helper function that can generate them for you. it'll make the code more readable:

```
func makeMap(key, value string) map[string]string {
  return map[string]string{ key: value }
}
```